README.md

```diff
@@ -7,8 +7,11 @@ Installation
 --------------------
 ```
 pipenv install -e "git+git://github.com/RedHatInsights/insights_connexion.git#egg=insights_connexion" -d
+pipenv install -e "git+git://github.com/fijshion/connexion.git@body_middleware_fix#egg=connexion"
```

Comment:
This should not be necessary. Once the [_body_middleware_fix_](https://github.com/fijshion/connexion/tree/body_middleware_fix) dependency is specified in [_setup.py_](
https://github.com/RedHatInsights/insights_connexion/pull/3/files#diff-2eeaed663bd0d25b7e608891384b7298), it grabs the patched version as a dependency.

Reply:
This is certainly not ideal. I spent an hour or so trying to get the dependency to install my fork instead of the master version. I didn't want to spend any more time on that since this is a temporary workaround. If you know a way to get the dependency working I'm all doing that instead.

Reply:
On my machine it worked just fine by setting a valid URL in [_setup.py_](
https://github.com/RedHatInsights/insights_connexion/pull/3/files/e8d6a9277f435603e73cd8c8bf2e6018e8b97d1c#diff-2eeaed663bd0d25b7e608891384b7298) with a VCS a version specified. But if this doesn’t work for you, I’d let it be and go your manual way for now.